ballerina-example-tool/README.md

```diff
@@ -0,0 +1,29 @@
+
+
+# 1. *build-bbe.sh*
+
+## This script will help to run the Ballerina in local machine.
+
+Requirements
+
+1. Clone ballerina-www to get the build-bbe.sh
```

Comment:
@sumangala028 Shall we modify this to build from current repository? We need to specify exact directory as well